src/applications/personalization/profile360/containers/PaymentInformationTOCItem.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { createIsServiceAvailableSelector } from 'platform/user/selectors';
+import backendServices from 'platform/user/profile/constants/backendServices';
+
+class PaymentInformationTOCItem extends React.Component {
+  static propTypes = {
+    isEligible: PropTypes.bool.isRequired,
+  };
+
+  render() {
+    if (!this.props.isEligible) {
+      return null;
+    }
+    return (
+      <li>
+        <a href="#direct-deposit">Direct deposit information</a>
```

Comment:
Going against my nature and totally NOT over-engineering this to start. So the link target and text are just hardcoded.